feats/selector.py

```diff
@@ -41,6 +47,28 @@ def serialize_data(self, app) -> dict:
         """
 
 
+class Default(Selector):
+    """
+    """
+    def __init__(self, feature):
+        super().__init__("Default")
+        self.feature = feature
+
+    def select(self, *args, **kwargs) -> str:
+        return self.feature.default_implementation.name
+
+    def used_implementation(self, impl: str, *args):
+        pass
+
+    # TODO: Separate out the distinction between a Selector and a user-created one.
+    @classmethod
+    def from_data(cls, app, configuration):
+        pass
+
+    def serialize_data(self, app) -> dict:
+        pass
```

Comment:
I assume this file is still in progress? or is this foundation for some future work?

Reply:
Nope, that's being used by this PR to simplify FeatureHandle's implementation. It could use the docstr being filled out.